app/views/items/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<p><%= link_to 'Merchants Index', '/merchants' %></p>
+<center><h2>Items</h2>
+<% @items.each do |item| %>
+  <h2><%= link_to item.name, "/items/#{item.id}" %></h2>
+  <p><%= item.description %></p>
+  <p>$<%= item.price %></p>
+  <p><%= image_tag item.image, alt: item.description %></p>
+  <p>Active? <%= item.status %></p>
+  <p>Quantity Available: <%= item.inventory %></p>
+  <p>Store: <%= Merchant.find(item.merchant_id).name %></p>
```

Comment:
Take this logic that you have on line 10 out, and instead use your active record association between items and merchants. `item.merchant` will return that merchant that sells the item.